chain/chain/src/store.rs

```diff
@@ -48,6 +48,32 @@ use crate::types::{Block, BlockHeader, LatestKnown, ReceiptProofResponse, Receip
 const CACHE_SIZE: usize = 100;
 const CHUNK_CACHE_SIZE: usize = 1024;
 
+lazy_static! {
+    pub static ref COL_GC: Vec<bool> = {
```

Comment:
Please rename into `SHOULD_COL_GC` or something more understandable from the first sight.